purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -30,6 +32,21 @@ data class PaywallData(
     @SerialName("default_locale") val defaultLocale: String,
     @SerialName("localized_strings") internal val localization: Map<String, LocalizedConfiguration>,
 ) {
+
+    /**
+     * @note This allows searching by `Locale` with only language code and missing region (like `en`, `es`, etc).
+     *
+     * @return [LocalizedConfiguration] for the given [Locale], if found.
+     */
+    fun configForLocale(requiredLocale: Locale): LocalizedConfiguration? {
+        return localization[requiredLocale.toString()]
+            ?: localization.takeIf { (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) }?.let { localization ->
+                localization.entries.firstOrNull { (localeKey, _) ->
+                    Locale(localeKey).isO3Language == requiredLocale.isO3Language
```

Comment:
Hmm is this a typo? Should it be `iso3Language`? If not, I wonder if we should use the `get` method, since this looks weird...

Reply:
haha I know it looks weird, but it's not a typo. I will use the `get` method yes